frontend/src/app/render_wasm.cljs

```diff
@@ -10,12 +10,13 @@
    [app.common.data.macros :as dm]
    [app.common.types.shape.impl :as ctsi]
    [app.common.uuid :as uuid]
-   [app.config :as cf]
+   [app.main.features :as features]
+   [app.main.store :as st]
    [app.util.object :as obj]
    [promesa.core :as p]))
 
 (def enabled?
-  (contains? cf/flags :render-wasm))
+  (features/active-feature? @st/state "render-wasm/v1"))
```

Comment:
This is no longer valid, because it can't be defined in this way. I mean having it as feature flag that can be altered by team, this variable will be set to true or false on the initialization, when team is still not loaded and will never change. 

This static var at namespace level is no longer the way to check for feature, the feature should be checked correctly where it is initialized and this var should not exist

Reply:
I have a separated task for handle workspace initialization + wasm renderer, so I guess we still can merge it as-is and address it in the separated task.